src/auth/googleauth.ts

```diff
@@ -632,7 +632,7 @@ export class GoogleAuth {
 
     // For GCE, return the service account details from the metadata server
     const {data} = await gcpMetadata.instance(
-        {property: 'service-accounts', params: {recursive: true}});
+        {property: 'service-accounts/', params: {recursive: true}});
```

Comment:
It'd be great to add some comments here that the slash at the end is important, and describes briefly what happened due to this, so that future maintainers make sure to not delete it.

Reply:
Great idea :) Added!